### ISSUE TITLE
[Suggestion] Protecting Item Frame Items

### ISSUE THREAD

**Issue by corexti** [Zrips/Residence#470]
So I noticed there isn't a way from Residence to protect the item frames. People can take out items and place on placed item frames inside a residence.

**Comment by Zrips** [comment]
item frames are protected, atleast they should be as they have been for quite some time now. Unless you have incorrect flag settup in your residence. If i'm not mistaken, residence needs to have container false flag

**Comment by corexti** [comment]
Yes I just checked and you're right. Residences must have container false and it doesn't work with 'removed' state. But is it not possible to put it on a separate flag (eg. frameprotect) so people can disable only that certain thing instead of disabling containers in general?

**Comment by Zrips** [comment]
But why would you want to remove protection from item frames? Adding extra flag isint really best option, especially when we already have quite few of them.

**Comment by corexti** [comment]
I meant adding protection for item frames, since people can take out items on default settings (container status undefined) and disabling containers will disable chests and other open-able blocks alongside with itemframes which is not practical. In my opinion item frames should be protected by default settings for a residence (I know it's configurable by disabling container default settings upon creation) and shouldn't merge with containers.

**Comment by tygh78071388** [comment]
> But why would you want to remove protection from item frames? Adding extra flag isint really best option, especially when we already have quite few of them.

really need separate the flags (item frame independent)
player can open chest but can't take out item on item frames.

This is very important for map art.

Please! thank you

**Comment by corexti** [comment]
> > But why would you want to remove protection from item frames? Adding extra flag isint really best option, especially when we already have quite few of them.
> 
> really need separate the flags (item frame independent) player can open chest but can't take out item on item frames.
> 
> This is very important for map art.
> 
> Please! thank you

That was the issue for me as well. People couldn't open chests, but they could steal items from item frames, which was the point of my issue and it was very annoying. Don't know if it's fixed now as I installed another plugin for item frames protection.

**Comment by tygh78071388** [comment]
> > > But why would you want to remove protection from item frames? Adding extra flag isint really best option, especially when we already have quite few of them.
> > 
> > 
> > really need separate the flags (item frame independent) player can open chest but can't take out item on item frames.
> > This is very important for map art.
> > Please! thank you
> 
> That was the issue for me as well. People couldn't open chests, but they could steal items from item frames, which was the point of my issue and it was very annoying. Don't know if it's fixed now as I installed another plugin for item frames protection.

What plugins have you installed? I've been looking for a long time...  1.17.1

**Comment by corexti** [comment]
> > > > But why would you want to remove protection from item frames? Adding extra flag isint really best option, especially when we already have quite few of them.
> > > 
> > > 
> > > really need separate the flags (item frame independent) player can open chest but can't take out item on item frames.
> > > This is very important for map art.
> > > Please! thank you
> > 
> > 
> > That was the issue for me as well. People couldn't open chests, but they could steal items from item frames, which was the point of my issue and it was very annoying. Don't know if it's fixed now as I installed another plugin for item frames protection.
> 
> What plugins have you installed? I've been looking for a long time... 1.17.1

I think it was called FrameProtect, although I'm not sure whether it worked normally because it hadn't been updated at the time.

**Comment by Baterka** [comment]
Armor-stands can be destroyed and ItemFrames looted in residence with both `destroy` and `container` set to false in v5.0.3.0 on 1.18.2 (and probably also on 1.19)

**Comment by Baterka** [comment]
@Zrips Armor-stands can be destroyed and ItemFrames looted in residence with both `destroy` and `container` set to false in v5.0.3.0 on 1.18.2 (and probably also on 1.19)

**Comment by VultarexCZ** [comment]
Any update on this? It would be really helpful.
